### ISSUE TITLE
Mockup good AA level link colors for all brands

### ISSUE THREAD

**Issue by wbuacct** [wherebyus/general-tasks#784]
Our link hover effects and the like aren't WCAG 2.1 AA compatible (inaccessible). So we need to design new ones.

## Reproduction Steps
> Ok. @schoeyfield if you want to throw out some good AA level link colors for all brands that’ll be good. I’m assuming a link is gonna be based on a brands primary color; we will address any other issues as they come I suppose

## Expected Results
Link hovers (and the like) that match WCAG 2.1 AA guidelines

## Actual Results
Link hovers (and the like) are currently not accessible

### Additional information:
 - BTID-395
 - [Link to Airtable](https://airtable.com/tblWM0skJCYIBSL6c/viwVfilkH4LzzRsKs/recFjig3vMvBkfcOw) - (may include image attachments)

### Submitted by
Name: Michael Schofield 
Email: michael@whereby.us
Slack ID: michael

**Comment by kassanj** [comment]
This ticket includes ticket #161 
https://app.zenhub.com/workspaces/tech-team-repos-5b3b863cf36bf13b2ad5b66e/issues/wherebyus/whereby.design/161
